Client/Frontend/Browser/BackForwardListViewController.swift

```diff
@@ -217,24 +221,31 @@ class BackForwardListViewController: UIViewController, UITableViewDataSource, UI
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-       let cell = self.tableView.dequeueReusableCell(withIdentifier: BackForwardListCellIdentifier, for: indexPath) as! BackForwardTableViewCell
+        let cell = self.tableView.dequeueReusableCell(withIdentifier: BackForwardListCellIdentifier, for: indexPath) as! BackForwardTableViewCell
         let item = listData[indexPath.item]
         let urlString = item.url.isLocal ? item.url.getQuery()["url"]?.unescape() : item.url.absoluteString
+        
+        cell.isCurrentTab = listData[indexPath.item] == self.currentItem
+        cell.connectingBackwards = indexPath.item != listData.count-1
+        cell.connectingForwards = indexPath.item != 0
+        cell.isPrivate = isPrivate
+        
         guard let url = urlString else {
-            return cell // This should never happen.
+            cell.site = Site(url: item.url.absoluteString, title: "Mozilla Firefox Start Page")
```

Comment:
Sorry for not noticing this earlier but since we'll want to translate this sentence for our different locales, we should move this string to `Strings.swift`. Check out how we've done some of the other strings to see how to add this one as well.